docker-compose.yml

```diff
@@ -205,6 +205,29 @@ services:
       KAFKA_LDAP_USER_NAME_ATTRIBUTE: uid
       KAFKA_LDAP_USER_OBJECT_CLASS: inetOrgPerson
 
+      # ConfluentTelemetryReporter: Disable
+      KAFKA_CONFLUENT_TELEMETRY_REPORTER_ENABLE: 'false'
+
+      # EmbeddedKafkaRest: Kafka Client Configuration
```

Comment:
To be explicit, can you please set the configuration parameter for the port for the embedded RP? On this and the broker below

Reply:
We can either set `KAFKA_CONFLUENT_METADATA_SERVER_LISTENERS` or `KAFKA_CONFLUENT_HTTP_SERVER` but not both. This is a consequence of MDS and KafkaRest sharing the same underlying HTTP server. The former only still exists for backwards compatibility reasons. I'm fine with fixing forward but this would make MDS less "explicit" so it's up to you .

Reply:
@rnpridgeon I'm confused then because the example in the docs that you added operates on endpoint `http://localhost:8082/v3/clusters/${KAFKA_CLUSTER_ID}/topics`, and 8082 appears to be the standalone RP, not the ERP?  The goal is to differentiate what action is being taken on which RP.

Reply:
Nevermind, standalone RP uses 8086, and the embedded RP uses 8082.  The confusion was resulting from the docker-compose output which shows 8082.  As a remediation, at the top of this RP section, it should state which port corresponds to which instance.

Reply:
Standalone RP doesn't share an underlying HTTP server instance with MDS; ERP does. It's not really a docker compose thing or a RestProxy thing it's an implementation detail with regard to how the RestProxy servlet is hosted. That said sure we can add a note at the top of the section